sql/hive/src/test/scala/org/apache/spark/sql/hive/client/VersionsSuite.scala

```diff
@@ -45,6 +45,8 @@ class VersionsSuite extends SparkFunSuite with Logging {
       Some(new File(sys.props("java.io.tmpdir"), "hive-ivy-cache").getAbsolutePath))
   }
 
+  private val mavenRepo = Some("http://www.datanucleus.org/downloads/maven2")
```

Comment:
Can you use `HIVE_METASTORE_MAVEN_REPO`'s default value at here? So, in case we update that default value, this one will be updated as well.